cmd/commands/parallel.go

```diff
@@ -2,17 +2,25 @@ package commands
 
 import "sync"
 
-func ApplyInParallel(names []string, action func(string) error) []error {
+type CorrelatedResult struct {
+	Input string
```

Comment:
In the future we may want this be type `interface{}` to allow for more than just strings. It's fine for today though.

Reply:
we may even have generics at that point, who knows? :D